cmd/juju/machine/add.go

```diff
@@ -341,3 +364,32 @@ func (c *addCommand) Run(ctx *cmd.Context) error {
 	}
 	return nil
 }
+
+func (c *addCommand) scopeRun(client AddMachineAPI, config *config.Config, ctx *cmd.Context) error {
```

Comment:
"scopeRun" doesn't really explain what it's doing. How about "maybeManuallyProvisionMachine"?